scripts/data_prep/refactor_dataset_text.py

```diff
@@ -0,0 +1,150 @@
+
```

Comment:
@tyoc213 the purpose of this script is to restructure the dataset for decontamination and for pytorch data collator right?